package-lock.json

```diff
@@ -2,291 +2,215 @@
   "requires": true,
```

Comment:
Please exclude changes made to the package-lock.json file from the PR, or specify why they're required.